install/configuring_inventory_file.adoc

```diff
@@ -473,6 +450,279 @@ xref:marking-masters-as-unschedulable-nodes[Configuring Schedulability on Master
 If set to `false, the default`, the Node Problem Detector is not installed or started.
 |===
 
+[[configuring-inventory-defining-node-group-and-host-mappings]]
+== Defining Node Groups and Host Mappings
+
+Starting in {product-title} 3.10, node configurations are now
+xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node-bootstrapping[bootstrapped]
+from the master. When the node boot and services are started, the node checks if
+a *_kubeconfig_* and other node configuration files exist before joining the
+cluster. If they do not, the node pulls the configuration from the master, then
+joins the cluster.
+
+This process replaces administrators having to manually maintain the node
+configuration uniquely on each node host. Instead, the contents of a node host's
+*_/etc/origin/node/node-config.yaml_* file are now provided by ConfigMaps
+sourced from the master.
+
+[[configuring-inventory--node-group-configmaps]]
+=== Node ConfigMaps
+
+The Configmaps for defining the node configurations must be available in the
+*openshift-node* project. ConfigMaps are also now the authoritative definition
+for node labels; the old `openshift_node_labels` value is effectively ignored.
+
+By default during a cluster installation, the installer creates the following
+default ConfigMaps:
+
+- `node-config-master`
+- `node-config-infra`
+- `node-config-compute`
+
+The following ConfigMaps are also created, which label nodes into multiple roles:
+
+- `node-config-all-in-one`
+- `node-config-master-infra`
+
+[IMPORTANT]
+====
+Changes should not be made to a node host's
+*_/etc/origin/node/node-config.yaml_* file. They can be overwritten by the
```

Comment:
s/can/will/

This it'll be overwritten about every 5 minutes or if the node service is restarted.